apps/bbsync/tests/test_query.py

```diff
@@ -1134,10 +1134,12 @@ def test_generate_hightouch_and_hightouch_lite(
         """
         Tests that major_incident_state is correctly converted into hightouch and
         hightouch-lite flags.
+        Other flag-producing fields are set not to produce flags for ease of testing.
```

Comment:
Changes of this type should be in a different commit. Also please see a comment in `test_generate_nist_cvss_validation()`, it might be useful.

Reply:
I'll move the change to a separate commit.